src/contribution.js

```diff
@@ -29,6 +29,12 @@ module.exports = (username, options) => {
     if (enableCors) url = `https://cors-anywhere.herokuapp.com/${url}`;
 
     https.get(url, response => {
+      if (response.statusCode !== 200) {
+        return reject({
```

Comment:
Should probably just reject with the whole response here, the extra information might be useful and it will stop ESLint complaining too 😄 

```suggestion
        return reject(response);
```